deploy/serverless-network/lambda-layers/all-dep/requirements.txt

```diff
@@ -0,0 +1,11 @@
+Flask==1.1.1
```

Comment:
We're using poetry now. We shouldn't ever have a requirements.txt file. Can you switch this to Poetry or be more specific about why we're doing PIP here?

Reply:
Done.